src/tracks/ui/SelectHandle.cpp

```diff
@@ -831,8 +831,8 @@ UIHandle::Result SelectHandle::Drag
    }
 
    if (evt.pCell) {
-      if ( auto clickedTrack =
-          static_cast<CommonTrackPanelCell*>(evt.pCell.get())->FindTrack() ) {
+      if (auto trackPanelCell = dynamic_cast<CommonTrackPanelCell*>(evt.pCell.get())) {
```

Comment:
Why the change?  Again because of that empty square above the ruler?

Reply:
The static_cast was justified by the fact that this UIHandle class is produced only by cell objects that are part of TrackPanel, and all other cells of a TrackPanel were guaranteed to be derived from CommonTrackPanelCell.